packages/vue/src/components/molecules/SfModal/SfModal.html

```diff
@@ -0,0 +1,17 @@
+<section class="sf-modal" :class="{ 'sf-modal--visible': visible }">
+  <SfOverlay class="sf-modal__overlay" :transition="transitionOverlay" v-show="visible" v-if="overlay" @click="checkPersistence"></SfOverlay>
```

Comment:
```suggestion
  <SfOverlay 
    class="sf-modal__overlay" 
    :transition="transitionOverlay" 
    v-show="visible" 
    v-if="overlay" 
    @click="checkPersistence"></SfOverlay>
```
please exlude to new line multiple property :) it's much easier to read <3 
https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended